src/main/scala/subsystem/HasTiles.scala

```diff
@@ -320,6 +334,12 @@ trait CanAttachTile {
     }
   }
 
+  /** Connect NMI inputs to the tile. No clock crossing needed. */
```

Comment:
why is no crossing needed? "Synchronization must be done prior to this connection"? or...

Reply:
done